utilities/ttl/db_ttl_impl.cc

```diff
@@ -309,5 +309,15 @@ Iterator* DBWithTTLImpl::NewIterator(const ReadOptions& opts,
   return new TtlIterator(db_->NewIterator(opts, column_family));
 }
 
+void DBWithTTLImpl::SetTtl(ColumnFamilyHandle *h, int32_t ttl) {
+    std::shared_ptr<TtlCompactionFilterFactory> filter;
+    Options opts;
+    opts = GetOptions(h);
+    filter = std::static_pointer_cast<TtlCompactionFilterFactory>(opts.compaction_filter_factory);
```

Comment:
line too long, please limit width to 80 characters.

Reply:
ok, will fix that.